framework/doc/content/source/interfacekernels/ADMatInterfaceReaction.md

```diff
@@ -0,0 +1,31 @@
+# ADMatInterfaceReaction
+
+## Description
+
+Specie M transports between two domains (domain 1 and domain 2), at the interface consider the following reaction is taking place:
+
+\begin{equation}
+M(1)\xrightleftharpoons[k_b]{k_f}M(2)
+\end{equation}
+
+With the first order reaction rate assuming a quasi-steady-state
+
+\begin{equation}
+\textrm{Reaction Rate} = \frac {\partial C_1} {\partial t} = k_f C_1 - k_b C_2 \approx 0
+\end{equation}
+
+where $C_1$ is the specie concentration in domain 1, $C_2$ is the specie concentration in domain 2, $k_f$ is the forward reaction coefficient, and $k_b$ is the backward reaction coefficient. `ADMatInterfaceReaction` object is used to impose this condition.
+
+[`InterfaceDiffusion`](/InterfaceKernels/index.md) is also used in this case to control flux at  the interface. However, the flux is not well-defined at boundaries. To address this issue, InterfaceReaction kernel applies a condition to constrain the discontinuity across the interface.
```

Comment:
> However, the flux is not well-defined at boundaries. To address this issue, InterfaceReaction kernel applies a condition to constrain the discontinuity across the interface.

There is a clear definition of the flux on + and - sides of interfaces and at boundaries. Perhaps you should say "The interface reaction kernel introduces a discontinuity in flux at the interface." ?

Reply:
Would "However, the flux is not [well-defined](https://en.wikipedia.org/wiki/Well-defined_expression) across the interface (potential discontinuity)" work?

I don't think we're really introducing a discontinuity as much as we are constraining it. Unless I'm mistaken, the `InterfaceDiffusion` interfacekernel only constrains the + and - side fluxes to be equal. In the absence of error, we would maintain the initial difference across the interface, but that's not actively being constrained unless some other physics is in play.

Reply:
I think the clearest documentation would be to write an equation describing the interface conditions being applied when both `ADMatInterfaceReaction` and `InterfaceDiffusion` are applied at the same interface